scalyr_agent/builtin_monitors/journald_monitor.py

```diff
@@ -415,7 +473,8 @@ def _process_entries(self):
             try:
                 msg = entry.get("MESSAGE", "")
                 extra = self._get_extra_fields(entry)
-                self._logger.emit_value("details", msg, extra_fields=extra)
+                logger = self.log_manager.get_logger(extra["unit"])
```

Comment:
Is `unit` guaranteed to always be present?  I think it would be safe to do a `extra.get("unit", "")` here just in case.